adversarial_workflow/evaluators/__init__.py

```
@@ -1,19 +1,45 @@
+    return evaluators
+
 
 __all__ = [
     "EvaluatorConfig",
     "EvaluatorParseError",
+    "run_evaluator",
+    "get_all_evaluators",
     "discover_local_evaluators",
     "parse_evaluator_yaml",
+    "BUILTIN_EVALUATORS",
 ]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)